cmake/Platform/Common/RuntimeDependencies_common.cmake

```diff
@@ -113,7 +113,7 @@ function(ly_get_runtime_dependencies ly_RUNTIME_DEPENDENCIES ly_TARGET)
             unset(target_locations)
             get_target_property(target_locations ${ly_TARGET} ${imported_property})
             if(target_locations)
-                list(APPEND all_runtime_dependencies ${target_locations})
+                list(APPEND all_runtime_dependencies "${target_locations}")
```

Comment:
found that python can produce issues because it has a `;` in the imported location within a generation expression